docs/resources/node_template.md

```diff
@@ -181,6 +181,19 @@ The following attributes are exported:
 * `tags` - (Optional) Comma-separated list of tags to apply to the Droplet (string)
 * `userdata` - (Optional) Path to file with cloud-init user-data (string)
 
+### `hetzner_config`
+
+#### Arguments
+
+* `api_token` - (Required) Hetzner Cloud project API token (string)
+* `image` - (Optional) Hetzner Cloud server image (string)
+* `server_location` - (Optional) Hetzner Cloud datacenter (string) 
+* `server_type` - (Optional) Hetzner Cloud server type (string)
+* `networks` - (Optional) Comma-separated list of network IDs or names which should be attached to the server private network interface (string)
+* `use_private_networks` - (Optional) Use private network (bool)
+* `volumes` - (Optional) Comma-separated list of volume IDs or names which should be attached to the server (string)
+* `userdata` - (Optional) Path to file with cloud-init user-data (string)
+
 ### `linode_config`
 
 #### Arguments
```

Comment:
As Hetzner node driver is not built in in Rancher by default, it should be installed and enabled before it can be used. Could you please add an full example on docs, about how to use it?? Something like https://github.com/rancher/terraform-provider-rancher2/pull/216#issue-358058877

Also, could you please add to the `hetzner_config` arguments, the default values that you set on `rancher2/schema_node_template_hetzner.go` ??

Reply:
Hi @rawmind0 
For the full example, can i add a prerequisites into the hetzner_config section in node_template.md or should i write this as a guide?

Reply:
I think you may add a comment on `hetzner_config` about requirement, and add an example on [node_template-example-usage section](https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/node_template#example-usage)

Something like this should be enough

```
# Create a new rancher2 Node Template using hetzner node_driver
resource "rancher2_node_driver" "hetzner" {
    active = true
    builtin = false
    description = "Rancher UI driver for the Hetzner Cloud docker driver."
    external_id = "https://github.com/mxschmitt/ui-driver-hetzner"
    name = "hetzner"
    ui_url = "https://storage.googleapis.com/hcloud-rancher-v2-ui-driver/component.js"
    url = "https://github.com/JonasProgrammer/docker-machine-driver-hetzner/releases/download/2.0.1/docker-machine-driver-hetzner_2.0.1_linux_amd64.tar.gz"
    whitelist_domains = ["storage.googleapis.com"]
}

resource "rancher2_node_template" "my_hetzner_node_template" {
  engine_install_url = "https://get.docker.com"
  name = "my_hetzner_node_template"
  driver_id = rancher2_node_driver.hetzner.id
  hetzner_config {
    api_token = "XXXXXXXXXX"
    image = "ubuntu-18.04"
    server_location = "nbg1"
    server_type = "cx11"
  }
}
```